.vscode/launchReactNative.js

```diff
@@ -0,0 +1,9 @@
+// This file is automatically generated by vscode-react-native@0.1.4
+// Please do not modify it manually. All changes will be lost.
+try {
+    var path = require("path");
+    var Launcher = require("/Users/sarahscott/.vscode/extensions/vsmobile.vscode-react-native-0.1.4/out/debugger/launcher.js").Launcher;
+    new Launcher(path.resolve(__dirname, "..")).launch();
+} catch (e) {
+    throw new Error("Unable to launch application. Try deleting .vscode/launchReactNative.js and restarting vscode.");
+}
```

Comment:
Exciting! Looking forward to hearing about the debugger experience.

Although it’s sad that they use absolute paths, because now this file is tied to your machine, even though they use relative paths on the second line :? Maybe this file isn’t actually re-generated often and we can just manually edit it to remove the absolute path?